common/app/common/NewNavigation.scala

```diff
@@ -382,21 +375,25 @@ object NewNavigation {
       SectionsLink("au/lifeandstyle/health-and-wellbeing", healthAu, Life)
     )
 
-    def getSectionLinks(sectionName: String, edition: Edition): Seq[NavLink] = {
+    def getSectionLinks(sectionName: String, edition: Edition): Tuple2[Seq[NavLink], Seq[NavLink]] = {
```

Comment:
Could this return `NavLinkLists` and if not then use the short-hand `(Seq[NavLink], Seq[NavLink])`?  Same below for the subsection function.